release/staging-build.yml

```diff
@@ -173,6 +173,9 @@ batch:
           CLONE_URL: https://git-codecommit.us-west-2.amazonaws.com/v1/repos/cilium.cilium
     - identifier: containerd_containerd_linux_amd64
       buildspec: buildspec.yml
+      depend-on:
+        - opencontainers_runc_linux_amd64
```

Comment:
this is odd, but it doesnt matter they will be both be built at the same time so it will not make a material difference.